src/PrintLoopNest.cpp

```diff
@@ -99,6 +98,15 @@ class PrintLoopNest : public IRVisitor {
             out << " in [" << min_val << ", " << max_val << "]";
         }
 
+        if (op->device_api != DeviceAPI::None) {
+          if (op->device_api == DeviceAPI::Host) {
```

Comment:
Hmm. I forget too, this was an old patch that has remained in my local repo for a long time. I'll check to see.